pydatalab/src/pydatalab/mongo.py

```diff
@@ -126,6 +144,14 @@ def create_default_indices(
 
     """
 
+    global ITEMS_FTS_FIELDS
+
+    if not ITEMS_FTS_FIELDS:
+        ITEMS_FTS_FIELDS = get_items_fts_fields()
+
+    if not ITEMS_FTS_FIELDS:
+        raise ValueError("Cannot create text indices: no fields available for full-text search")
```

Comment:
This is super error prone and hard to test -- better to just call the function everywhere